contributors/devel/running-locally.md

```diff
@@ -25,7 +25,7 @@ Getting started locally
 
 #### Linux
 
-Not running Linux? Consider running [Minikube](http://kubernetes.io/docs/getting-started-guides/minikube/), or on a cloud provider like [Google Compute Engine](../getting-started-guides/gce.md).
+Not running Linux? Consider running [Minikube](http://kubernetes.io/docs/getting-started-guides/minikube/), or on a cloud provider like [Google Compute Engine](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/getting-started-guides/gce.md).
```

Comment:
https://github.com/kubernetes/kubernetes.github.io/blob/master can be replaced by kubernetes.io, so the link becomes:
kubernetes.io/docs/getting-started-guides/gce.md